ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -211,20 +217,15 @@ const Footer = ({ isFormValid, setIsFormValid, setStepNotification, isInProgress
                                       trigger="manual"
                                       isVisible={!isFormValid}
                                     />}
-                                <Button
-                                  variant="secondary"
-                                  isDisabled={isBackDisabled}
-                                  onClick={onBack}>
-                                    {_("Back")}
-                                </Button>
                                 <Button
                                   id="installation-quit-btn"
+                                  style={ { marginLeft: "var(--pf-global--spacer--2xl)"  } }
                                   variant="link"
                                   onClick={() => {
                                       setQuitWaitsConfirmation(true);
                                   }}
                                 >
-                                    {_("Quit")}
+                                    {_("Cancel")}
```

Comment:
Why was this button changed to "Cancel"? It's obvious that "Quit" would quit the installer, but it's not so obvious in case of "Cancel".

Reply:
Good catch! Maybe we just pulled it with the layout change from PatternFly by mistake ?

Reply:
:thinking: Sorry I thought the Quit button should have been changed as well. I misunderstood the screenshot on issue: INSTALLER-3361.

![image](https://user-images.githubusercontent.com/40278421/216313930-7f9bfa74-b1ec-4bfc-814f-52255934f2d1.png)

Reply:
Thanks & sorry for the confusion. :)